ansible_collections/arista/cvp/molecule/cv_container_v3/test_cv_container_v3_configlet.yaml

```diff
@@ -0,0 +1,191 @@
+  hosts: CloudVision
+  connection: local
+  gather_facts: false
+  collections:
+    - arista.cvp
+  vars:
+
+    ansible_command_timeout: 1200
+    ansible_connect_timeout: 600
+
+    CVP_CONFIGLETS:
+      configlet1: '! This is first configlet'
+      configlet2: '! This is second configlet'
+
+    TOPOLOGY_3_TIER:
+      TEAM01:
+        parentContainerName: Tenant
+      TEAM01_DC:
+        parentContainerName: TEAM01
+      TEAM01_LEAFS:
+        parentContainerName: TEAM01_DC
+        configlets:
+          - configlet1
+
+    CHANGE_CONFIGLET_LOOSE:
+      TEAM01_LEAFS:
+        parentContainerName: TEAM01_DC
+        configlets:
+          - configlet2
+
+    CHANGE_CONFIGLET_STRICT:
+      TEAM01_LEAFS:
+        parentContainerName: TEAM01_DC
+        configlets:
+          - configlet1
+
+    REMOVE_CONFIGLET:
+      TEAM01_LEAFS:
+        parentContainerName: TEAM01_DC
+
+  tasks:
+    # Create configlet
+
+    - name: "Push config"
+      arista.cvp.cv_configlet_v3:
+          configlets: "{{CVP_CONFIGLETS}}"
+          state: present
+      register: CV_CONFIGLET
+
+    # TC1 - Create and delete 3 tier topology.
+    # Testing: Removal of entire topology.
+
+    - name: Build Container Topology On {{inventory_hostname}}
+      arista.cvp.cv_container_v3:
+        topology: '{{TOPOLOGY_3_TIER}}'
+      register: Build_TOPO_3_TIER
+
+    - name: Testing Build Container
+      assert:
+        that:
+          - Build_TOPO_3_TIER.container_added.changed == true
+          - Build_TOPO_3_TIER.container_added.container_added_count == 3
+          - Build_TOPO_3_TIER.container_added.container_added_list == ["TEAM01", "TEAM01_DC", "TEAM01_LEAFS"]
+          - Build_TOPO_3_TIER.container_added.success == true
+
+    - name: Remove Container Topology On {{inventory_hostname}}
+      arista.cvp.cv_container_v3:
+        topology: '{{TOPOLOGY_3_TIER}}'
+        state: absent
+      register: REMOVE_TOPO_3_TIER
+
+    - name: Testing Remove Container
+      assert:
+        that:
+          - REMOVE_TOPO_3_TIER.container_deleted.changed == true
+          - REMOVE_TOPO_3_TIER.container_deleted.container_deleted_count == 3
+          - REMOVE_TOPO_3_TIER.container_deleted.container_deleted_list == ["TEAM01_LEAFS", "TEAM01_DC", "TEAM01"]
+          - REMOVE_TOPO_3_TIER.container_deleted.success == true
+
+    # TC2 - Create a 3 tier topology and attach configlet to one of the containers and remove it.
+    # Testing:
+    #       1. Adding configlet
+    #       2. Change in configlet with apply_mode: loose
+    #       3. Change in configlet with apply_mode: strict
+    #       4. Remove configlet using apply_mode: strict
+    #       5. Removal of container which contains configlet.
+
+    - name: Build Container Topology On {{inventory_hostname}} With Configlet
+      arista.cvp.cv_container_v3:
+        topology: '{{TOPOLOGY_3_TIER}}'
+      register: CONFIGLET_ADD
+
+    - name: Testing Added Configlet
```

Comment:
Can we get configlet facts from CVP and check if it's applied on the container we want?

Reply:
Added!